kin-sdk-core/src/androidTest/assets/testConfig.json

```diff
@@ -30,5 +30,5 @@
  "private_key":"0xabf8c2dd52f5b14ea437325854048e5daadbca80f99f9d6f8e97ab5e05d4f0ab"
  }
 ], 
-"token_contract_address":"0x8919486b0afaad4656e8f9667ce9adbb62c3f2b1"
+"token_contract_address":""
```

Comment:
we can add this file into .gitignore

Reply:
I will get rid of this files soon when I get to tests

Reply:
👍  thanks